frontend/src/lib/components/LemonInput/LemonInput.scss

```diff
@@ -17,25 +17,27 @@
             opacity: var(--opacity-disabled);
             cursor: not-allowed;
         }
+
+        &:placeholder-shown {
+            font-style: italic;
+        }
+    }
+
+    &.LemonRow {
+        padding: 0.25rem 0.5rem;
     }
 }
 
 .LemonRow__icon:first-child {
     .LemonInput & {
-        color: var(--muted);
-    }
-    .LemonInput.LemonInput--with-input & {
-        color: var(--primary);
+        .LemonIcon {
+            color: var(--muted-alt);
+        }
     }
-}
-
-.LemonInput--embedded {
-    color: var(--default);
-    font-weight: 400;
-    border: none !important;
-
-    .LemonRow__icon {
-        color: var(--muted-alt);
+    .LemonInput.LemonInput--hasContent & {
+        .LemonIcon {
+            color: var(--primary);
+        }
```

Comment:
Decided to take care of updating the remaining styles to match how they are documented in the Design System